mono/mini/debugger-agent.h

```diff
@@ -31,6 +31,22 @@ struct _MonoDebuggerCallbacks {
 };
 
 typedef struct _DebuggerTlsData DebuggerTlsData;
+typedef enum _MonoDebuggerThreadState MonoDebuggerThreadState;
```

Comment:
You could just declare the enum here.

Reply:
I can do that, but @kumpera in channel sounded like he wanted to make these changes another way. If this is still destined for merge, I'm happy to make that change.